failsafe-controller/src/main/java/net/jodah/failsafe/package-info.java

```diff
@@ -0,0 +1,445 @@
+ * controller in lieu of Failsafe. To better understand how this is done, let's look at how one
+ * normally invokes Failsafe with an example:
+ *
+ * <p><code><pre>
+ *     Future&lt;MyObject&gt; future = Failsafe.with(myRetryPolicy)
+ *       .with(myScheduledExecutor)
+ *       .onRetry(this::logFailure)
+ *       .onAbort(this::logInterruptionAndRecreate)
+ *       .onFailure(this::logAndRecreateIfNotCancelled)
+ *       .onSuccess(this::logAndSetCreated)
+ *       .get(this::create);
+ * </pre></code>
+ *
+ * <p>In this example, the first line will create a {@link net.jodah.failsafe.SyncFailsafe} object.
+ * The second line will create a {@link net.jodah.failsafe.AsyncFailsafe} object from it and the
+ * next 4 lines registers event listeners. Additional lines could be added to further configure
+ * Failsafe if required. The final line is the one that actually triggers the start of an execution
+ * which returns a {@link java.util.concurrent.Future} allowing access to the result at the end and
+ * also provides the ability to cancel the execution.
+ *
+ * <p>The Failsafe controller is not a replacement for Failsafe but does require replacing the first
+ * line above. One way to achieve this is by defining a variable in your class that can be injected
+ * through the constructor at build time. This variable could be defined as a function like this:
+ *
+ * <p><code>
+ * private final Function&lt;RetryPolicy, SyncFailsafe&lt;MyObject&gt;&gt; createFailsafeCreator;
+ * </code>
+ *
+ * <p>With a package private constructor like this:
+ *
+ * <p><code><pre>
+ *     MyClass(Function&lt;RetryPolicy, SyncFailsafe&lt;MyObject&gt;&gt; createFailsafeCreator) {
+ *       this.createFailsafeCreator = createFailsafeCreator;
+ *     }</pre></code>
+ *
+ * <p>The normal constructor could simply call that constructor like this:
+ *
+ * <p><code>this(Failsafe::with)</code>
+ *
+ * <p>From this point the example above would be re-written as:
+ *
+ * <p><code><pre>
+ *     Future&lt;MyObject&gt; future = createFailsafeCreator.apply(myRetryPolicy)
+ *       .with(myScheduledExecutor)
+ *       .onRetry(this::logFailure)
+ *       .onAbort(this::logInterruptionAndRecreate)
+ *       .onFailure(this::logAndRecreateIfNotCancelled)
+ *       .onSuccess(this::logAndSetCreated)
+ *       .get(this::create);
+ * </pre></code>
+ *
+ * <p>While testing, the constructor would be called as follow:
+ *
+ * <p><code><pre>
+ *     FailsafeController controller = new FailsafeController("some name");
+ *
+ *     MyClass mine = new MyClass(controller::with);
+ * </pre></code>
+ *
+ * <p>From this point on, the <code>controller</code> instance becomes your point of contact to
+ * control, mock, or script how Failsafe will respond to each executions and attempts.
+ *
+ * <p><b>Actions</b>
+ *
+ * <p>Each time Failsafe makes an attempt, the controller intercepts the call to the task specified
+ * (see 7th line in the above example) and figures out what to do. This could be:
```

Comment:
✏️ I would remove the "see 7th line" parenthetical. If you want to clarify, then simply note that this is when the method under test is invoked to return the `Future`.